modules/user/src/main/JsonView.scala

```diff
@@ -22,8 +22,8 @@ final class JsonView(isOnline: lila.socket.IsOnline) {
         "createdAt" -> u.createdAt
       )
       .add("disabled" -> u.disabled)
-      .add("tosViolation" -> u.marks.engine)
-      .add("booster" -> u.marks.boost)
+      .add("tosViolation" -> (u.marks.engine || u.marks.boost))
+      .add("booster" -> false) // App BC
```

Comment:
I believe the app will be fine if this field is omitted.

Reply:
thanks I missed [that](https://github.com/ornicar/lila/blob/6048f3c4e223357ea99ed84ea4f0a82f251eb2fe/modules/common/src/main/base/PimpedJsObjects.scala#L50) 😄